help/en/html/doc/Technical/Help.shtml

```diff
@@ -69,29 +80,90 @@
         ones are stored in the <a href="WebSite.shtml">main web
         site</a>)</li>
       </ul>
-      In the long run, we want only two branches to this
+      
+      <p>While this is how we would LIKE our help files organized, help pages are created
+      by members of the community and sometimes general descriptions are put in the package
+      directory rather than the html directory and vice versa. [If you see something that 
+      needs to be moved, please post a note to the jmri-developers online groups.io forum
+      as there are members who occassionally try to do some housekeeping.]</p>
+      
+      <p></p>In the long run, we want only two branches to the help file
       structure: the "package" part of the tree for help
       information that is specific to a particular piece of code,
       and another part of the tree for more general information.
       (It's a religious issue whether that 2nd part should be
       called "html" or "manual"; the key thing is we end up with
       just one). The web is meant to be a <em>web</em>, with many
       paths through it to reach content. The second part of the
-      tree should also be organized as "one page, one topic", with
+      tree ("html") should also be organized as "one page, one topic", with
       links to connect them as needed.
+      
+             <p><a href="#helpTop">[Go to top of page]</a></p>
+       </div>
+ 
+
+<!-- = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = --> 
+      <a name="web" id="web"></a>
+      <h2>Web Access to Help Pages</h2>
+
+    <div class="para">      
+
+      It's inconvenient to have to maintain two separate web pages for the main web site and the help
+      system. To reduce the need for that, we use a particular form for the web pages in the help system.
+      [Additional information can also be found <a href="WebSite.shtml">here.</a>]
+
+      <ul>
+        <li>Use ".shtml" file extensions so that the main web
+        servers will search the files for <a href=
+        "http://httpd.apache.org/docs/1.3/howto/ssi.html">server-side
```

Comment:
Please update to https://httpd.apache.org/docs/1.3/howto/ssi.html

Reply:
Change made. Thanks.